src/app/work-item/work-item-quick-add/work-item-quick-add.component.scss

```diff
@@ -1,6 +1,6 @@
-@import "../../../assets/stylesheets/variables";
-@import "../../../assets/stylesheets/list-group";
-@import "../../../assets/stylesheets/list-view";
+@import "./../../../assets/stylesheets/variables";
+@import "./../../../assets/stylesheets/list-group";
+@import "./../../../assets/stylesheets/list-view";
```

Comment:
Was the location wrong or did the folder get moved?
Is this related to this PR?